shared/templates/sudo_defaults_option/ansible.template

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Ensure {{{ OPTION }}} is enabled in /etc/sudoers
+  lineinfile:
+    path: /etc/sudoers
+    regexp: '^(?!#)[\s]*Defaults.*[\s]+{{{ OPTION }}}[\s]*.*$'
```

Comment:
A couple of questions:
- Why is there `(?!#)` at the beginning? Wouldn't it be the same if it was removed?
- The `[\s]*` by the end should probably be `[\s]+`, as `[\s]*.*` is the same as `.*`

Reply:
Well noted, I have removed the negative lookahead, and changed the check for whitespace for word boundary.

Reply:
You have fixed it in OVAL, but it's still present in Bash/Ansible, and those rule files have another form of the regex in OCIL as well.